src/middlewares/jwtAuth.ts

```diff
@@ -12,7 +12,10 @@ function jwtAuth(req: Request, res: Response, next: NextFunction): any {
     if (!authToken) {
       throw new ApplicationError(401, "UnAuthorized Access");
     }
-    const token = authToken.split(" ")[1];
+    const [bearer, token] = authToken.split(" ");
+    if (bearer != "Bearer") {
+      throw new ApplicationError(400, "Invalid Token");
+    }
```

Comment:
Check for token, not for bearer

Reply:
Again Message is wrong, It should be "Token not found". When you verify the token and it does not matches, then It will be Invalid token.